hedera-node/src/main/java/com/hedera/services/fees/calculation/utils/OpUsageCtxHelper.java

```diff
@@ -111,7 +115,8 @@ public ExtantFeeScheduleContext ctxForFeeScheduleUpdate(TokenFeeScheduleUpdateTr
 	public ExtantCryptoContext ctxForCryptoUpdate(TransactionBody txn) {
 		final var op = txn.getCryptoUpdateAccount();
 		ExtantCryptoContext cryptoContext;
-		var info = workingView.infoForAccount(op.getAccountIDToUpdate(), aliasManager);
+		var info = workingView.infoForAccount(
```

Comment:
This is another strong point in favor of tracking the # of associations in `MerkleAccountState`---you are now potentially making 1000 `MerkleMap.get()` calls just to compute the fee of a `CryptoUpdate`. 😲